src/schemas/friendship.ts

```diff
@@ -23,6 +23,13 @@ export enum FriendshipSceneType {
   QRCode   = 30,
 }
 
+export interface FriendshipSource{
+  sourceContactId ?:string;
+  sourceName ?:string;
+  shareCardContactId?:string;
+  shareCardName?:string;
+}
```

Comment:
In your example, the `sourceContactId` is the same as the `shareCardContactId`.

![image](https://user-images.githubusercontent.com/23626907/104856843-a32f3300-594f-11eb-9439-6afd9f30352c.png)

Could you please:

1. add those example data values as comments in your code, so the interface will be easy to be understood by others
2. add an explanation for the differentiation between `sourceXXX` and `shareXXX`?

Reply:
ok

Reply:
```sourceXXX``` should be id/name who share the card. 
```shareXXX``` maybe always empty string, no similar results have been obtained since that snapshot...